tpl/aws/bootstrap.sh.erb

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+# Apply Terraform IaC
+pushd terraform
+terraform apply
+BASTION_IP=$(terraform output bastion_ip)
+popd
+
+# Establish an SSH tunnel to Bastion
+ssh -D 5000 -fNC ubuntu@$BASTION_IP -i <%= @values['kite']['private_key_path'] %>
+export BOSH_ALL_PROXY=socks5://localhost:5000
+
+# Render BOSH manifest and related files
+kite render-manifest bosh
+
+# Create new BOSH environment
+bosh create-env bosh_director.yml --state=state.json --vars-store=creds.yml --vars-file=bosh_vars.yml --var-file private_key=<%= @values['kite']['private_key_path'] %> -o cpi.yml
+
+# Configure alias for the new environment
+bosh alias-env <%= @values['bosh']['name'] %> -e <%= @values['bosh']['static_ip'] %> --ca-cert <(bosh int ./creds.yml --path /director_ssl/ca)
+
+# Login to newly created BOSH Director
+export BOSH_CLIENT=admin
+export BOSH_CLIENT_SECRET=`bosh int ./creds.yml --path /admin_password`
+
+# Query the Director for more info
+bosh -e <%= @values['bosh']['name'] %> env
+
+# Render Concoure and related files
```

Comment:
I would prefer you move all of this in the README instead
For now this script should only take care of bosh